odoo/modules/registry.py

```diff
@@ -433,8 +434,7 @@ def check_signaling(self):
             # Check if the model caches must be invalidated.
             elif self.cache_sequence != c:
                 _logger.info("Invalidating all model caches after database signaling.")
-                self.clear_caches()
-                self.cache_invalidated = False
+                self.clear_caches(invalidate=False)
```

Comment:
This API looks confusing to me.  Method `clear_caches` invalidates the cache, but what does it mean with `invalidate=False`?

What about naming the parameter `signal`?
```python
    registry.clear_caches(signal=True)   # invalidate the cache and signal it to other workers
    registry.clear_caches(signal=False)  # invalidate the cache locally only
```

Reply:
Or maybe `propagate`?
```python
registry.clear_caches(propagate=True)   # invalidate the cache and propagate invalidation
registry.clear_caches(propagate=False)  # invalidate the cache locally only
```

Reply:
You're right, `signal` seems good to me.

I'll make the changes.